sys/conf/kern.mk

```diff
@@ -270,12 +270,15 @@ CFLAGS+=	-gdwarf-2
 #
 .if ${MACHINE_ABI:Mpurecap}
 
+.if defined(CHERI_SUBOBJECT_BOUNDS)
+CFLAGS+=	-Xclang -cheri-bounds=${CHERI_SUBOBJECT_BOUNDS}
 .if defined(CHERI_SUBOBJECT_BOUNDS_STATS)
```

Comment:
Shouldn't this be non-nested? We still get CSetBounds for alloca's that contribute to stats IIRC.

Reply:
This just matches bsd.cheri.mk which was then copied into bsd.cpu.mk.  We should keep the subobject bounds bits consistent for userland and kernel I think.